src/scripts/main.js

```diff
@@ -1,3 +1,292 @@
+      personsRender = sortedAlphabet(personsNow);
+      break;
+
+    case 'Age':
+      if (counterClickAge > 0) {
+        personsRender = personsNow.reverse();
+        break;
+      }
+
+      resetCount();
+      counterClickAge++;
+
+      personsRender = personsNow.sort((a, b) => {
+        return +a.children[whereСlicked.cellIndex].innerText
+        - +b.children[whereСlicked.cellIndex].innerText;
+      });
+      break;
+
+    case 'Salary':
+      if (counterClickSalary > 0) {
+        personsRender = personsNow.reverse();
+        break;
+      }
+
+      resetCount();
+      counterClickSalary++;
+
+      personsRender = personsNow.sort((a, b) => getSalary(a
+        .children[whereСlicked.cellIndex].innerText)
+        - getSalary(b.children[whereСlicked.cellIndex].innerText));
+  }
+
+  tableBody.append(...personsRender);
+});
+
+tableBody.addEventListener('click', (e) => {
+  [...document.querySelectorAll('.active')]
+    .forEach(el => el.classList.remove('active'));
+
+  e.target.closest('tr').classList.add('active');
+});
+
+buttonForm.onclick = (e) => {
+  e.preventDefault();
+
+  const labels = [...document.querySelectorAll('label')];
+
+  let valueInput = [];
+
+  valueInput = labels.map(label => [...label.children][0].value);
+
+  if (valueInput.some(item => item === '')
+  || valueInput[0].length < 4
+  || valueInput[3] <= 18
+  || valueInput[3] >= 90
+  ) {
+    body.insertAdjacentHTML('afterend', `
+    <div class="notification error" style="position: fixed">
+      <h2>Error</h2>
+      <p>Fields filled out incorrectly. Umya must have 4 
+      characters, age must be greater than 18 and less than 90</p>
+      </br>
+    </div>
+  `);
+
+    setTimeout(() => document.querySelector('.notification').remove(), 5000);
+
+    return;
+  }
+
+  const newTableElement = `
+    <tr>
+      <td>${valueInput[0]}</td>
+      <td>${valueInput[1]}</td>
+      <td>${valueInput[2]}</td>
+      <td>${valueInput[3]}</td>
+      <td>${'$' + (+valueInput[4]).toLocaleString()}</td>
+    </tr>
+  `;
+
+  tableBody.insertAdjacentHTML('afterbegin', newTableElement);
+
+  body.insertAdjacentHTML('afterend', `
+  <div class="notification success" style="position: fixed">
+    <h2>Well done</h2>
+    <p>You have added a new employee</p>
+    </br>
+  </div>
+`);
+
+  setTimeout(() => document.querySelector('.notification').remove(), 2000);
+
+  labels.map(label => {
+    [...label.children][0].value = '';
+
+    return label;
+  });
+};
+
+let elementText;
+
+window.addEventListener('dblclick', (e) => {
+  if (e.target.closest('tbody') === null) {
+    return;
+  }
+
+  if (e.target.closest('tbody').classList.contains('table-body')) {
+    elementText = e.target.innerText;
+
+    e.target.closest('td').innerHTML = `
+    <input 
+      class="input-edit" 
+      value="${elementText}"
+      style="
+        outline:none;
+        background: none;
+        font-size:15px; 
+        height: 28px;
+        width: 100px;
+      "
```

Comment:
try to style in way that table wouldn't change it's width after you've inserted field